### ISSUE TITLE
What would be a good TTA to add? 

### ISSUE THREAD

**Issue by kentaroy47** [kentaroy47/ODA-Object-Detection-ttA#5]

**Comment by piantic** [comment]
Hi, I saw your notebook in kaggle.

After the GWD competition, I was writing a similar way for tta in my private repo.
So I am interested in this repo. :)

I used two types of Rotate90.
- RotateLeft90
- RotateRight90

How about add this?

If you like this, I will pull request later.

**Comment by kentaroy47** [comment]
Yes, this will be a good thing to add. Would be great if you can make a PR
on this too! Thanks for the suggestion 😊

2020年11月14日(土) 3:31 piantic <notifications@github.com>:

**Comment by piantic** [comment]
When the business work is organized, I will PR on that part(maybe this week).
:)
